layouts/partials/head.html

```diff
@@ -19,7 +19,6 @@
     as="style"
   />
 {{ end }}
-<link rel="stylesheet" href="/pagefind/pagefind-ui.css" />
```

Comment:
The styling of the search seems to be broken, maybe that's because of some missing styles that were in the CSS before? 

<img width="1249" height="262" alt="Image" src="https://github.com/user-attachments/assets/65feacf6-f337-4132-bfa0-0911d9bf8b0a" />